recipes/default.rb

```diff
@@ -32,11 +32,32 @@
   mode 00755
 end
 
-execute 'install homebrew' do
-  command homebrew_go
-  environment lazy { { 'HOME' => ::Dir.home(homebrew_owner), 'USER' => homebrew_owner } }
-  user homebrew_owner
-  not_if { ::File.exist? '/usr/local/bin/brew' }
+begin
+  template '/etc/sudoers.d/homebrew_sudo' do
```

Comment:
Mavericks... isn't that a little out dated and thus insecure to be used for anything?

But I do see your point, I've only done testing on 'new' OS X versions.